tests/network/bindingplugin.go

```diff
@@ -56,16 +53,9 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 
 	Context("passt", func() {
 		BeforeEach(func() {
-			err := libkvconfig.WithNetBindingPlugin(passtBindingName, v1.InterfaceBindingPlugin{
-				SidecarImage:                passtSidecarImage,
-				NetworkAttachmentDefinition: passtNetAttDef,
-			})
-			Expect(err).NotTo(HaveOccurred())
-		})
-
-		BeforeEach(func() {
+			Expect(libkvconfig.RegisterPasstPlugin(libnet.PasstNetAttDef)).To(Succeed())
```

Comment:
I probably commented about this somewhere else already: I do not think this change is needed.